cuda_pathfinder/cuda/pathfinder/README.md

```diff
@@ -0,0 +1,46 @@
+# `cuda.pathfinder` Module
+
+## Public API for loading NVIDIA Dynamic Libs
+
+* `cuda.pathfinder.SUPPORTED_NVIDIA_LIBNAMES` (`tuple[str]`)
+
+* `cuda.pathfinder.load_nvidia_dynamic_lib(libname: str) -> LoadedDL`
+
+* `cuda.pathfinder.LoadedDL`:
+    * `handle` (platform-specific type)
```

Comment:
Any concern with this being platform-specific type and API stability? Is this effectively the handle returned from `dlopen` on Linux?

Reply:
Yes, for Linux it's a plain `int`, for Windows a `pywintypes.HANDLE`. This seems to be mainstream for each platform, and very unlikely to require changes.

I guess we could introduce a level of indirection (some wrapper type), but I figure that'll be far more trouble (confused users) than we could hope to avoid.

Reply:
Agreed on the wrapper type. Last question since this is public API is is this `handle` needed to be a public attribute today? Do we expect usage outside of it how we use it for `cuda.bindings`?